docs/source/getting_started/python-guide.rst

```diff
@@ -13,7 +13,46 @@ To use Iroha Python library, you need to get it from the
 
 	pip3 install iroha
 
-Now, as we have the library, we can start sending the actual transactions.
+Creating your own key pairs with Python library
+-----------------------------------------------
+
+For testing purposes, you can use example keys.
+But you can also create your own.
+To create **native Iroha encrypted** keys (difference between algorithms can be found `here <../develop/keys.html>`__), please use the following code:
+
+.. code-block:: python
+
+	from iroha import IrohaCrypto
+
+	# these first two lines are enough to create the keys
+	private_key = IrohaCrypto.private_key()
+	public_key = IrohaCrypto.derive_public_key(private_key)
+
+	# the rest of the code writes them into the file
+	with open('keypair.priv', 'wb') as f:
+	    f.write(private_key)
+
+	with open('keypair.pub', 'wb') as f:
+	    f.write(public_key)
+
+And for HL Ursa ed25519 SHA-2 keys in Multihash format, please use:
+
+.. code-block:: python
+
+	from iroha import IrohaCrypto
+	import iroha.ed25519_sha2 as ed25519
+
+	private_key, _ = ed25519.create_keypair()
+	public_key = IrohaCrypto.derive_public_key(private_key)
+
+	with open('keypair.priv', 'wb') as f:
+	    f.write(private_key)
+
+	with open('keypair.pub', 'wb') as f:
+	    f.write(public_key)
```

Comment:
with respect to [5c1c0f4](https://github.com/hyperledger/iroha-python/pull/54/commits/5c1c0f436b91a433e53a710fbbf24b3698b015c3),

```suggestion
	import binascii

	private_key = SigningKey.generate()
	public_key = private_key.verify_key
	
	with open('keypair.priv', 'wb') as f:
	    f.write(binascii.hexlify(private_key.encode()))
	    
	with open('keypair.pub', 'wb') as f:
	    f.write(binascii.hexlify(public_key.encode()))
```